### PR TITLE
Use proposed drawableTouching interface from render

### PR DESCRIPTION
**Pull request by gnarf**
### Resolves

* [x] Requires https://github.com/LLK/scratch-render/pull/325
* [ ] Would finish/fix https://github.com/LLK/scratch-render/issues/284

### Test Coverage

will add tests

### CODE REVIEW

**Comment by gnarf** [comment]
This just needs vm/gui to update scratch-render package now

Tests were added in the render module.

**Comment by gnarf** [comment]
cc @cwillisf - this is finishing up that other PR I landed

**Review by cwillisf — APPROVED** [review]

**Comment by cwillisf** [comment]
LGTM, assuming it builds and works once the render changes are merged... ;)

**Pull request merged**
